educational-core/src/com/jetbrains/edu/learning/codeforces/courseFormat/CodeforcesCourse.kt

```diff
@@ -39,7 +39,7 @@ class CodeforcesCourse : Course {
   override fun isViewAsEducatorEnabled(): Boolean = false
 
   fun getContestUrl(): String = getContestURLFromID(id)
-  fun getSubmissionUrl(): String = "${getContestUrl()}/${CODEFORCES_SUBMIT}?locale=$languageCode"
+  fun getSubmissionUrl(task: CodeforcesTask? = null): String = "${getContestUrl()}/${CODEFORCES_SUBMIT}?locale=$languageCode${if (task == null) "" else "&submittedProblemIndex=" + task.name.substringBefore(".")}"
```

Comment:
Code formatting: this line has more than 210 symbols, but the maximum is 140. Could you please correct this?